content/docs/studio/user-guide/team-collaboration/index.md

```diff
@@ -179,22 +177,10 @@ seats. This means that you can invite 2 more collaborators. At this point, if
 you remove any one team member or pending invite, that seat becomes available
 and so you will have 3 remaining seats.
 
-### Change your team plan and team size
+## Get Enterprise
```

Comment:
There were a few sections that mostly repeated this. We should be able to cut down to just this.